examples/nextjs-linear-like-issue-tracker/src/components/Editor.tsx

```diff
@@ -76,17 +76,20 @@ export function Editor({
                 {contentFallback}
               </div>
             ) : (
-              <RichTextPlugin
-                contentEditable={
-                  <ContentEditable className="outline-none editor-styles" />
-                }
-                placeholder={
-                  <div className="absolute top-0 left-0 pointer-events-none text-neutral-500 whitespace-nowrap">
-                    Start typing here…
-                  </div>
-                }
-                ErrorBoundary={LexicalErrorBoundary}
-              />
+              <>
+                <RichTextPlugin
+                  contentEditable={
+                    <ContentEditable className="outline-none editor-styles" />
+                  }
+                  placeholder={
+                    <div className="absolute top-0 left-0 pointer-events-none text-neutral-500 whitespace-nowrap">
+                      Start typing here…
+                    </div>
+                  }
+                  ErrorBoundary={LexicalErrorBoundary}
+                />
+                <EditorFloatingToolbar />
```

Comment:
Since the issue is fixed in the latest version, we can move this back outside of the conditional.